src/context/AppContext.js

```diff
@@ -33,36 +33,53 @@ export function useAppContext() {
 }
 
 export default function AppContextProvider({ children }) {
-  const [storedTodo, setStoredTodo] = useLocalStorage("todo", []);
-  const [todoList, todoDispatch] = useReducer(todoReducer, storedTodo);
+  // const [storedTodo, setStoredTodo] = useLocalStorage("todo", []);
+  const [todoList, todoDispatch] = useReducer(todoReducer, []);
   const [pending, setPending] = useState([]);
   const [paused, setPaused] = useState([]);
   const [completed, setCompleted] = useState([]);
```

Comment:
Instead of having multiple states you can make use of useReducer and
 initialState={
todoList:[],
pending:[],
completed:[]
}